README.md

```diff
@@ -451,13 +454,13 @@ If you want the benchmarks to run using a specific build, or the `cairo-run` com
 sierra2mlir program.sierra -o program.mlir
 
 # translate mlir to llvm-ir
-"$MLIR_SYS_160_PREFIX"/bin/mlir-translate --mlir-to-llvmir program.mlir -o program.ll
+"$MLIR_SYS_170_PREFIX"/bin/mlir-translate --mlir-to-llvmir program.mlir -o program.ll
```

Comment:
I think we should add the `mlir-opt` step too before `mlir-translate` because we're not lowering to the LLVM dialect automatically anymore.

An example on how to run the command may be found in `scripts/bench-hyperfine.sh`.